### ISSUE TITLE
Significant events data

### ISSUE THREAD

**Issue by zbraniecki** [covidatlas/coronadatascraper#392]
There's a number of significant events that affect data either directly or in delayed fashion and may impact projections. Things like:

* Social distancing imposed
* Shelter-in-place imposed
* Lockdown imposed
* Testing policy change (only performed when result would alter treatment)
* Mass masks use recommended/advised/required

**Comment by caticoa3** [comment]
Yes! Is there a website that can be scraped for this, its seems like this would need to be done more manually. Most social distancing is not neatly organized in tables like the Covid counts. Instead news reports about when measure were put in place would need to be aggregated.

**Comment by lazd** [comment]
This is tough information to gather and I think is currently outside of the scope of this project.

HOWEVER, if there is a source we're currently scraping that provides this data, I would accept a PR to add it into the dataset (though I'm not sure what that would look like).

I'll leave this issue open in case someone finds data and needs an issue to associate it with.

**Comment by caticoa3** [comment]
School closures:
https://www.edweek.org/ew/section/multimedia/map-coronavirus-and-school-closures.html

**Comment by zbraniecki** [comment]
> This is tough information to gather and I think is currently outside of the scope of this project.

Right, I understand that. It may be a good material for a separate repository to accumulate those events manually. I just think that the same community as this one would be interested in that data :)

**Comment by zbraniecki** [comment]
* closing borders
* cancelling mass gatherings above X people
* cancelling domestic flights

**Comment by saralioness** [comment]
This spreadsheet has a number of data points for all US states and territories in one document: https://www.nga.org/wp-content/uploads/2020/03/CoronavirusStateActionsChart_26March2020.xlsx

Just to name a few: 

- Emergency declaration
- National Guard activation
- Statewide school closures
- Stay-at-home orders

They update this spreadsheet daily and change the link/filename to match the current day. 

Page source: https://www.nga.org/coronavirus/#federal
(At-a-Glance section directly below JHU Dashboard, link to spreadsheet and PDF)

**Comment by caticoa3** [comment]
Heads up: currently, it looks like only the latest 4 weekdays are available, and anything before or on the 24th March 2020 is not available.

**Comment by shaperilio** [comment]
Summary of 50 states:
https://www.usatoday.com/story/news/nation/2020/03/30/coronavirus-stay-home-shelter-in-place-orders-by-state/5092413002/
